plugins/ibm_qradar/plugin.spec.yaml

```
@@ -0,0 +1,997 @@
+      type: integer
+      description: "Offense type"
+      required: false
+    policy_category_count:
+      title: "Policy Category Count"
+      type: integer
+      description: "Policy category count"
+      required: false
+    protected:
+      title: "Protected"
+      type: boolean
+      description: "Protected"
+      required: false
+    relevance:
+      title: "Relevance"
+      type: integer
+      description: "Relevance"
+      required: false
+    remote_destination_count:
+      title: "Remote Destination Count"
+      type: integer
+      description: "Remote destination count"
+      required: false
+    rules:
+      title: "Rules"
+      type: "[]rules"
+      description: "Rules"
+      required: false
+    security_category_count:
+      title: "Security Category Count"
+      type: integer
+      description: "Security category count"
+      required: false
+    severity:
+      title: "Severity"
+      type: integer
+      description: "Severity"
+      required: false
+    source_address_ids:
+      title: "Source Address IDs"
+      type: "[]integer"
+      description: "Source address IDs"
+      required: false
+    source_count:
+      title: "Source Count"
+      type: integer
+      description: "Source count"
+      required: false
+    source_network:
+      title: "Source Network"
+      type: string
+      description: "Source network"
+      required: false
+    start_time:
+      title: "Start Time"
+      type: integer
+      description: "Start time"
+      required: false
+    status:
+      title: "Status"
+      type: string
+      description: "Status"
+      required: false
+    username_count:
+      title: "Username Count"
+      type: integer
+      description: "Username count"
+      required: false
+actions:
+  start_ariel_search:
+    title: Start Ariel Search
+    description: Start an Ariel search
+    input:
+      aql:
+        title: AQL Query
+        type: string
+        description: AQL query to perform the search
+        required: true
+        example: Select * from events
+    output:
+      data:
+        title: Search Data
+        type: search
+        description: JSON data for the Search
+        required: false
+  get_ariel_search_by_id:
+    title: Get Ariel Search by ID
+    description: Get an ariel search by ID
+    input:
+      search_id:
+        title: Search ID
+        type: string
+        required: true
+        description: ID of the Ariel search to get
+        example: 9102cb1d-5994-4f78-8f08-16c6f6991015
+      poll_interval:
+        type: number
+        required: false
+        description: Number of seconds to recheck until the search is completed
+        default: 1
+        example: 1
+    output:
+      data:
+        title: Search Data
+        type: search
+        description: JSON data of the Search
+        required: false
+  get_assets:
+    title: Get Assets
+    description: List all assets found in the model
+    input:
+      range:
```

Extending [Mike's](https://github.com/rapid7/insightconnect-plugins/pull/1057/files#r787301324) comment, several of the input and output parameters across this plugin's actions are missing the `title` value. The `title` value specifies what text describes this variable's text field in the InsightConnect product user interface. Most of the time it is the same as the section label for this parameter, but it should be formally capitalized and punctuated.
For example, in this input parameter 'range` on line 740, please add this line:
```suggestion
      range:
        title: Range
```
Remember to regenerate the plugin after making these changes.